Casks/reminders-menubar.rb

```diff
@@ -0,0 +1,28 @@
+cask "reminders-menubar" do
+  version "1.17.0"
+  sha256 "e3fdf46b3fc339bde71a14296f544c74ca099058d3845e86aca0d2fdcf8557fe"
+
+  url "https://github.com/DamascenoRafael/reminders-menubar/releases/download/v#{version}/reminders-menubar.zip"
+  name "Reminders MenuBar"
+  desc "Simple menu bar app to view and interact with reminders"
+  homepage "https://github.com/DamascenoRafael/reminders-menubar"
+
+  depends_on macos: ">= :big_sur"
+
+  app "Reminders MenuBar.app"
+
+  postflight do
+    application = "#{appdir}/Reminders MenuBar.app"
```

Comment:
Not all users install the apps in #{appdir}, this should cover more use cases. 
```suggestion
    application = "#{staged_path}/#{token}"
```

Reply:
Thanks, I hadn't thought about the possibility of installations outside #{appdir}.